.gitignore

```diff
@@ -47,3 +47,4 @@ node_modules
 *.code-workspace
 
 .savant/cache
+astro/.astro/types.d.ts
```

Comment:
per their doc I think we can do that whole directory. eg. "astro/.astro/". `types.d.ts` is the only thing in there now but Astro's language makes it sound like they might dump other stuff in there one day